.github/workflows/continuous-integration-workflow.yml

```diff
@@ -233,6 +233,8 @@ jobs:
           DOCS_SSH_KEY: ${{ secrets.DOCS_SSH_KEY }}
           DOCS_HOST: ${{ secrets.DOCS_HOST }}
   perform_release:
+    permissions:
```

Comment:
This file doesn't seem to have the default permissions set at the top. Should they be?

Reply:
I can't determine what permissions are required at the top level.

Reply:
Sorry for the delay, I was on break. Ok, thanks for the update, we can proceed with what you have here and go from there.

Reply:
👍